core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/SdxClusterShape.java

```diff
@@ -0,0 +1,8 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.common;
+
+public enum SdxClusterShape {
```

Comment:
as this must match with the other `SdxClusterShape` it must be ensured via unit tests